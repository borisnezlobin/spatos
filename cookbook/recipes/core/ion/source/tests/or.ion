false 
or true && echo "false or true"

true 
or true && echo "true or true"

false 
or false && echo "false or false"

if or true
    echo "if: or true"
else 
    echo "else: or true"
end

echo '# ANCHOR: or_demonstration'
test "42" = "4"
or test 4 == 4 && echo "Will be printed: At least one of the two commands returns successfully"
test "42" = "4"
or test 4 = 2 && echo "Will not be printed: None of the two commands returns successfully"
echo '# ANCHOR_END: or_demonstration'
