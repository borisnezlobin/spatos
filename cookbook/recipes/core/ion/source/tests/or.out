false or true
true or true
if: or true
# ANCHOR: or_demonstration
Will be printed: At least one of the two commands returns successfully
# ANCHOR_END: or_demonstration
